cli/src/plugins/javadoc/buildIndexes.ts

```diff
@@ -28,15 +35,55 @@ export const buildIndexes = async ({
           }
           const link = linkToEntity as TypedNode<"link">;
           return toctreeItem({
-            url: link.url.replace(/#.*$/, ""),
-            value: entity.canonicalName,
+            url: packageToFolderPath(link.url.replace(/#.*$/, "")),
+            value: entity.canonicalName.replace(/#.*$/, ""),
           });
         })
         .filter((e) => e !== undefined) as ToctreeItemNode[]
     ),
+    brk,
+    list("unordered", pkgs.map((pkg) => {
+      return md.listItem(finalizedProject.linkToEntity(pkg.canonicalName))
+    }))
   ]);
   finalizedProject.writePage({
     path: "/index",
     root,
   });
+
+  const packages = finalizedProject.entities
+    .filter((e) => e.data?.category === "package")
+
+  packages.forEach(pkg => {
+    const members = finalizedProject.entities
+      .filter((e) => e.data?.category === "class")
+      .filter((e) => e.data?.containingPackage === pkg.canonicalName)
+
+    const memberClasses = members
+      .filter((e) => e.data?.classType === "class").sort((a, b) => { return a.canonicalName.localeCompare(b.canonicalName) } );
```

Comment:
No need for the returns here. I understand wanting to repeat it instead of creating a standalone function though as you would have to repeat the type if so.
```suggestion
      .filter((e) => e.data?.classType === "class").sort((a, b) => a.canonicalName.localeCompare(b.canonicalName));
```